### DISCUSSION TITLE
Clarification on Sailing / low charge start limit

### DISCUSSION THREAD

**Discussion by zaxxon72** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#435] [General]
Hi,

I don't know whether I understand sailing correctly, and there not being a trial mode, I cannot try before buy.

Say I want to charge my battery to 80% at max. And when I'm on mains, this will not go down. For the sake of the argument, let's say sailing is set to 20%.
What happens if I disconnect mains and work for 5%, 15%, 25%, and then connect back to mains?

Will it start charging from 75% to 80? from 65% to 80%? from 55% to 80%?

What I would like to have (and expect) is that when disconnecting and reconnecting (even several times), charging would not resume until I hit 60%. And then charge to 80%.
Or does sailing only apply when connected the 'first time', and 

Timeline: 80% - disconnect- work until 71% - reconnect - power from mains, no charging- disconnect at 71% - work until 65% - reconnect - power from mains, no charging- disconnect at 65% - work until 55% - reconnect, charge until 80% (- as bonus scenario, continuation: deconnect after charging to 71% - run from battery to 67% - reconnect - charge to 80% - power from mains, no charging) 

thanks a lot for clearing this up
simon

**Comment by MatthiasKerbl** [comment]
Hi @zaxxon72,

Thank you for your interest in AlDente Pro.

> Will it start charging from 75% to 80? 

No

> from 65% to 80%?

No

> from 55% to 80%?

Yes

> What I would like to have (and expect) is that when disconnecting and reconnecting (even several times), charging would not resume until I hit 60%. And then charge to 80%.

That’s exactly how the Sailing Mode currently works.

> Timeline: 80% - disconnect- work until 71% - reconnect - power from mains, no charging- disconnect at 71% - work until 65% - reconnect - power from mains, no charging- disconnect at 65% - work until 55% - reconnect, charge until 80%

You can expect this behavior.

> - as bonus scenario, continuation: deconnect after charging to 71% - run from battery to 67% - reconnect - charge to 80% - power from mains, no charging

Currently, in this scenario, Sailing Mode would not continue charging after reconnecting at 67%. Instead, it would pause charging until you hit 60% or lower.

Hope it clarifies a couple of things.

Cheers,
Matthias
